.buildkite/gpu-tests/scripts/run-tests.sh

```
@@ -0,0 +1,81 @@
+  uv pip install coverage pytest-cov https://github.com/neuralmagic/pytest-nm-releng/archive/v0.4.0.tar.gz
+  FLAGS_FILE="coverage_flags.sh"
+  nmre-generate-coverage-flags --package "llmcompressor" --output-file "${FLAGS_FILE}"
+  source "${FLAGS_FILE}"
+  rm "${FLAGS_FILE}"
+  export PYTEST_ADDOPTS
+fi
+
+echo "--- Installing compressed-tensors (nightly)"
+git clone --quiet https://github.com/vllm-project/compressed-tensors.git
+uv pip uninstall compressed-tensors
+export GIT_CEILING_DIRECTORIES="$(pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)